src/panel/components/Pane.tsx

```diff
@@ -73,13 +86,19 @@ const PaneRoot: FC<ComponentProps<typeof PaneContainer>> = ({
   );
 
   return (
-    <PaneContainer {...props} style={{ ...props.style, ...style }}>
+    <PaneContainer
+      {...props}
+      style={{ ...props.style, ...style }}
+      data-portrait={`${isPortrait}`}
+      ref={paneRef}
+    >
       {children}
       <DraggingEdge
         role="seperator"
         aria-orientation={isPortrait ? "horizontal" : "vertical"}
         aria-grabbed={grabbed}
         onMouseDown={handleClick}
+        data-portrait={`${isPortrait}`}
```

Comment:
I'd probably stick to media queries for this, usually I use data attributes which are specific to the element (i.e. element color, element active, etc, rather than viewport status)